core/domain/question_domain.py

```diff
@@ -82,7 +82,9 @@ def __init__(self, change_dict):
 
         if self.cmd == CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION:
             self.question_dict = change_dict['question_dict']
-            self.skill_id = change_dict['skill_id']
+            self.skill_id = None
```

Comment:
Shouldn't we be passing "skill_id: null" explicitly in the change dict for this case? Each type of change_dict should have a fully-determined set of keys.

Reply:
Ok will do!

Reply:
Done. At creation, I am passing skill_id: None, in the change_dict.